EnglishFilter/sections/foreign.txt

```diff
@@ -8741,6 +8741,7 @@ mm-cg.com##.TW_UTtoy
 !------------------------------ Thai ---------------------------------
 !---------------------------------------------------------------------
 !
+*.gif$domain=hanimesubth.com,image
```

Comment:
can be added without content type as `.gif^$domain=hanimesubth.com`

Reply:
Well, it makes sense at least on uBO. Anyway it's a tiny difference, not?

Reply:
How do you measure the impact on performance?

Reply:
I don't rigorously measure, but gorhill benchmarks regularly and we uAssets maintainer generally care about tokenization. See gorhill's reply here: `https://www.reddit.com/r/uBlockOrigin/comments/lovx3z/what_is_the_cleanest_way_to_block_an_image_format/`
Ofc it is AG repo here, but some people including me use Base filter on uBO too ;)

Reply:
Yep, thanks @Yuki2718. I based on that because uBO isn't bad after all.